ballerina-release-notes-0.990.0.md

```diff
@@ -390,6 +390,130 @@ Final variables
 Regular variables
 structure members
 
+## Endpoints, Services and Listeners
+
+### Endpoints
+
+In Ballerina, endpoints represent an external system that one or more workers interact with. There are two types of endpoints, Listener endpoints or client endpoints. 
+
+Previously, both these endpoints were modeled using two object definitions. One defined the life cycle of the endpoint, other defined the Actions of the endpoint. Additionally, endpoint keyword was used to represent endpoint instances and it had its own initialization and assignment syntax.
+
+With this release, the endpoints are declared as client objects. With this change following are removed from the language. 
+- endpoint declaration eg: `endpoint http:Listener helloWorldEP { port : 9090 }`
```

Comment:
```suggestion
- endpoint declaration e.g., `endpoint http:Listener helloWorldEP { port : 9090 }`
```

Normally "for example" is specifically written as "e.g." as it's a latin abbreviation and needs some specific punctuation for the abbreviation to be correct.

Reply:
Ack.